### PR TITLE
GUI-2760 limit protocol choices when logged into AWS

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-2760

### CODE REVIEW

File: eucaconsole/static/js/widgets/listener-editor/listener-editor.js [hunk 1]
```diff
@@ -42,7 +48,7 @@ angular.module('ELBListenerEditorModule', ['ModalModule'])
 
             this.portsValid = function () {
                 var fromValid = this.sourceValid(vm.from);
-                var toValid = this.targetValid(vm.to);
+                var toValid = (vm.to != null)?this.targetValid(vm.to):true;
```
Comment (kamalgill) [inline eucaconsole/static/js/widgets/listener-editor/listener-editor.js]: This line could be simplified to the following (also note added white space for readability):
```
var toValid = !!vm.to ? this.targetValid(vm.to) : true; 
```

**Comment by aliceh** [comment]
Retest this, please

**Comment by aliceh** [comment]
Retest this, please

**Comment by aliceh** [comment]
Retest this, please

**Comment by aliceh** [comment]
Retest this, please

**Comment by aliceh** [comment]
There is a problem with the CI cloud. I am working on the issue

**Comment by aliceh** [comment]
Retest this please

**Pull request merged**